test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -42,7 +42,14 @@ std::vector<int> get_available_cpus() {
     CPU_ZERO(mask);
 
     int ret = sched_getaffinity(0, sizeof(cpu_set_t), mask);
-    UT_ASSERTeq(ret, 0);
+
```

Comment:
pls describe in commit message what's really changed here - it seems like changing UT_ASSERTs is again, just a small addition 😉

Reply:
done?

Reply:
heh, now I'm reading the commit message and see that you do 2 changes (change functions and fix segfault) in one commit - usually we tend to do 1 change per commit - e.g. in case you need to revert one of this changes for some reason;

please split this into 2 commits

Reply:
Done?

Reply:
nope, there's still a little confusion in the commits:
- they both edit the same place (one commit introduced some change and the other one overrides them with some fixes),
- the title and description should describe what's inside of the commit - more or less it should be one thing

that said, I believe we can leave this for dessert; When all changes are done you can split the commits (or, if that's easier, you can start from scratch).

Reply:
Done? 🥺